plugins/MCWhitelist/whitelist_http_api.py

```diff
@@ -0,0 +1,24 @@
+import json
+import requests
+
+from Globals import mongo_client, WHITELIST_HTTP_API_BEARER_TOKEN
+
+WHITELIST_HTTP_API_BASE = "http://craft.muncompsci.ca:7500"
+HEADERS = {"authorization": f"WHA {WHITELIST_HTTP_API_BEARER_TOKEN}"}
+
+
+class WhitelistHttpApi:
+    def whitelist(self):
+        return requests.request("GET", WHITELIST_HTTP_API_BASE, headers=HEADERS).json()
+
+    def add(self, username):
+        data = json.dumps({"name": username})
+        requests.request(
+            "POST", WHITELIST_HTTP_API_BASE, data=data, headers=HEADERS
```

Comment:
You can use the `json` argument rather than the `data` argument so you don't need to handle stringifying your json yourself